demon.js

```diff
@@ -186,6 +186,56 @@ function observeImpfstoff() {
   setTimeout(observeImpfstoff, rateLimit);
 }
 
+let availabilities = false
+function observePunctumMedico() {
+  if (!availabilities) {
+    log('checking Punctum Medico')
+
+    FormData = require('form-data')
+    const formData = new FormData();
+    formData.append('uniqueident', '5a72efb4d3aec');
+
+    axios.post(
+      'https://onlinetermine.zollsoft.de/includes/searchTermine_app_feature.php',
+      formData,
+      {
+        headers: formData.getHeaders()
+      }
+    ).then(function (response) {
+      response?.data?.termine.forEach(function (appt) {
+        if (!appt.length) {
+          return;
+        }
+
+        const matchingAppt = appt.some((detail) => {
+          if (!detail) {
+            return
+          }
+
+          const lowercase = detail.toLowerCase()
+          return lowercase.includes('erstimpfung')
+        })
+
+        if (!matchingAppt) {
+          return;
+        }
+
+        log("Punctum Medico success", appt);
+
+        availabilities = true;
+        setTimeout(function () {
+          availabilities = false;
+        }, 60000);
+
+        notify();
+      });
+    })
+    .catch(error);
+  }
+
+  setTimeout(observePunctumMedico, rateLimit);
```

Comment:
I would move this to the first line of the function. If any error happens during the function execution it won't be rescheduled.